particula/__init__.py

```diff
@@ -1,4 +1,4 @@
-""" a simple, fast, and powerful particle simulator.
+""" A simple, fast, and powerful particle simulator.
```

Comment:
Btw, these lowercase edits were inspire by the following:

```
>>> import particula
>>> help(particula)
Help on package particula:

NAME
    particula - a simple, fast, and powerful particle simulator.

DESCRIPTION
    particula is a simple, fast, and powerful particle simulator,
    or at least two of the three, we hope. It is a simple particle
    system that is designed to be easy to use and easy to extend.
    The goal is to have a robust aerosol (gas phase + particle phase)
    simulation system that can be used to answer scientific questions
    that arise for experiments and research discussions.
    
    The main features of particula are:
    ...
    
    More details to follow.

PACKAGE CONTENTS
    aerosol_dynamics (package)

DATA
    u = <pint.registry.UnitRegistry object>

VERSION
    0.0.3

FILE
    .../python3.9/site-packages/particula/__init__.py

(END)
```

In particular, see

```
NAME
    particula - a simple, fast, and powerful particle simulator.
```

So this line often appears after the name of the file (module, etc.) and a dash. I thought it would look nicer if they were all lowercase to start. But this is just being silly on my part. Also, the little space between the `"""` and the top line is to make sure indentation is the same (`"""` = 3 spaces, and our Python indentation is 4 spaces)

Eventually, I want to figure out a way to automate making these docstrings and comments into actual documentation in [here](https://uncscode.github.io/particula/documentation/d000_detailed_documentation.html)

Reply:
Thanks for explaining that! It makes way more sense now that it is lower case because it comes after a dash. I can revert those changes.